OpenChart.Tests/src/Formats/OpenChart/Data/TestProjectData.cs

```diff
@@ -0,0 +1,41 @@
+using NUnit.Framework;
+using OpenChart.Formats.OpenChart.Version0_1.Data;
+using System;
+
+namespace OpenChart.Tests.Formats.OpenChart.Data
+{
+    public class TestProjectData
+    {
+        [Test]
+        public void Test_Init()
+        {
+            var data = new ProjectData();
+            Assert.IsEmpty(data.Charts);
+            Assert.NotNull(data.Metadata);
+        }
+
+        [Test]
+        public void Test_Validate_NullMetadata()
+        {
+            var data = new ProjectData();
+
+            // I don't know why this would ever happen but ¯\_(ツ)_/¯
+            data.Metadata = null;
+
+            try
+            {
+                data.Validate();
+                Assert.Fail("Expected Validate method to throw an exception.");
```

Comment:
`Assert.Throws` only works if the exception you give it is an exact match. All this test cares about is that Validate is throwing an exception, it doesn't really care what that exception is (since there's no restrictions on what kinds of exceptions Validate should throw)

Reply:
In that case, how about `Assert.Catch`, which allows derived exception types, and then you can just assert on the base exception class?